### PR TITLE
Updated Order and Refund Events

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

**Comment by clintonb** [comment]
We should use Segment's semantic events to avoid future confusion:

* https://segment.com/docs/spec/ecommerce/v2/
* https://segment.com/docs/integrations/google-analytics/

**Comment by clintonb** [comment]
@edx/ecommerce @edx/helio please review

**Comment by clintonb** [comment]
Google Tag Manager has been updated to recognize the "Order Completed" event in addition to "Completed Order" and "Completed Purchase".

File: ecommerce/static/js/pages/receipt_page.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ define([
         'use strict';
 
         function trackPurchase(order_id, total_amount, currency) {
-            window.analytics.track('Completed Order', {
+            window.analytics.track('Order Completed', {
```
Comment (mjfrey) [inline ecommerce/static/js/pages/receipt_page.js:12]: Do we need to tell Adley about this change?

**Review by mjfrey — APPROVED** [review]

File: ecommerce/static/js/pages/receipt_page.js (hunk 1 above)
Comment (clintonb) [reply ecommerce/static/js/pages/receipt_page.js:12]: I have no idea.

Comment (mjfrey) [reply ecommerce/static/js/pages/receipt_page.js:12]: I would just let BA know about the change in case it affects the reporting.

**Comment by douglashall** [comment]
@clintonb What needs to be updated in GA/GTM in order to support this change? There are WL containers and properties in play that we need to consider.

File: ecommerce/static/js/pages/receipt_page.js (hunk 1 above)
Comment (clintonb) [reply ecommerce/static/js/pages/receipt_page.js:12]: Agreed. I have notified them via HipChat.

**Comment by clintonb** [comment]
@douglashall I made the following changes in GTM:

1. Renamed the "Completed Purchase" trigger to "Order Completed" (not required, but good for consistency).
2. Added a regex for the event name: `Completed (Order|Purchase)|Order Completed`

**Pull request merged**